rebac-admin-backend/v1/identities.go

```diff
@@ -0,0 +1,176 @@
+// Copyright 2024 Canonical Ltd.
+// SPDX-License-Identifier: Apache-2.0
+
+package v1
+
+import (
+	"encoding/json"
+	"net/http"
+
+	"github.com/canonical/identity-platform-admin-ui/rebac-admin-backend/v1/resources"
+)
+
+// GetIdentities returns the list of known identities.
+// (GET /identities)
+func (h handler) GetIdentities(w http.ResponseWriter, req *http.Request, params resources.GetIdentitiesParams) {
+	identities, err := h.Identities.ListIdentities(&params)
+	if err != nil {
+		response := h.IdentitiesErrorMapper.MapError(err)
+		writeResponse(w, response.Status, response)
+		return
+	}
+
+	response := resources.GetIdentitiesResponse{
+		Data:   identities.Data,
+		Status: 200,
```

Comment:
Let's use `http.Status*` constants instead of integer literals.

Reply:
Yes, this is taken care of in the next PR with the missing endpoints from this one
Thanks!